service/storage_service.cc

```diff
@@ -2544,15 +2552,83 @@ class topology_coordinator {
         , _raft(raft_server), _term(raft_server.get_current_term())
         , _raft_topology_cmd_handler(std::move(raft_topology_cmd_handler))
         , _tablet_allocator(tablet_allocator)
+        , _load_stats_refresh_trigger([this] {
+            // Awakes the topology event handler, which allows stats to be refreshed
+            // if coordinator allows for handling of tablet migrations.
+            _topo_sm.event.signal();
+        })
         , _ring_delay(ring_delay)
     {}
 
     future<> run();
 };
 
+future<locator::load_stats> topology_coordinator::refresh_table_load_stats() {
+    locator::load_stats stats;
+
+    auto tm = get_token_metadata_ptr();
+    auto& topology = tm->get_topology();
+
+    for (auto& [dc, endpoints] : topology.get_datacenter_endpoints()) {
+        locator::load_stats dc_stats;
+        slogger.info("raft topology: Refreshing table load stats for DC {} that has {} endpoints", dc, endpoints.size());
+        for (auto& endpoint : endpoints) {
+            auto* node = topology.find_node(endpoint);
+            if (!node) {
+                // TODO: that shouldn't happen, perhaps abort instead.
+                continue;
+            }
+            auto dst = node->host_id();
+
+            auto node_stats = co_await ser::storage_service_rpc_verbs::send_table_load_stats(&_messaging,
+                netw::msg_addr(id2ip(dst)),
+                _as,
+                raft::server_id(dst.uuid()));
```

Comment:
Should send those rpcs in parallel.

Reply:
What about failures? If a node is down or we just time out (what's the timeout?), we shouldn't fail.

Up to a certain failure rate we can just extrapolate, above we should avoid making decisions, but not stop the caller.